pkg/skaffold/build/local/types.go

```diff
@@ -75,12 +76,18 @@ func NewBuilder(runCtx *runcontext.RunContext) (*Builder, error) {
 		pushImages = *runCtx.Cfg.Build.LocalBuild.Push
 	}
 
+	tryImportMissing := false
+	if runCtx.Cfg.Build.LocalBuild.TryImportMissing != nil {
+		tryImportMissing = *runCtx.Cfg.Build.LocalBuild.TryImportMissing
```

Comment:
`TryImportMissing` should probably just be of type bool since it defaults to `false`

Reply:
@tejal29 asked him to turn it into a pointer.  A pointer at least allows us to later check if the value was explicitly specified.

Reply:
What for?

Reply:
If we change the default?

Reply:
I see. Not sure we should take such future changes into account. pointers on booleans are the worst